web/docusaurus.config.js

```diff
@@ -234,7 +234,7 @@ module.exports = {
           },
           {
             from: '/docs/tutorials/todo-app',
-            to: '/docs/tutorial/project-structure',
+            to: '/docs/tutorial/create',
```

Comment:
Linking to the first page of the tutorial since we want to link to the tutorial, not a specific page.